pkg/containers/client.go

```diff
@@ -171,6 +193,35 @@ func (c *Client) addContainerWithCgroup(container containerdContainers.Container
 		PIDs:         pids,
 	}
 
+	sandbox, err := c.criRuntimeServiceClient.ListPodSandbox(ctx, &criapi.ListPodSandboxRequest{
+		Filter: &criapi.PodSandboxFilter{
+			Id: container.SandboxID,
+		},
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(sandbox.Items) > 0 {
+		for k, v := range sandbox.Items[0].Labels {
+			if _, ok := c.forwardedLabels[k]; ok {
```

Comment:
I think it would be beneficial to also support wildcards.

Reply:
As minimum it should contain prefix

Reply:
With prefix - simple map lookup won't work, will need to match with all labels slice